t/venom/vars/all.yml

```diff
@@ -671,3 +671,9 @@ firewall_sso.radius.radius_user_name: foo
 firewall_sso.radius.radius_user_password: bar
 firewall_sso.radius.radius_mock_dir: '{{.pfserver_root_dir}}/t/mock_servers/node_radius'
 firewall_sso.radius.type: FortiGate
+
+#################################################################################
+## html e2e tests
+#################################################################################
+html.pfappserver_baseurl: 'https://{{.pfserver_mgmt_ip}}:1443'
```

Comment:
You can use directly: `{{.pfserver_webadmin_url}}`

This is the value used by default in `t/venom/lib/html_e2e.yml`

Reply:
I didn't want to hardcode this in the vars since the tests may eventually need to work on the captive-portal, or some other host, port or path.